diff --git a/src/io/mhdf/CMakeLists.txt b/src/io/mhdf/CMakeLists.txt
index 258701e..0aa39da 100644
--- a/src/io/mhdf/CMakeLists.txt
+++ b/src/io/mhdf/CMakeLists.txt
@@ -18,20 +18,20 @@ set(

 add_library( mhdf OBJECT ${mhdf_LIB_SRCS} )

-add_executable( h5minfo include/mhdf.h example/info.c $<TARGET_OBJECTS:mhdf> )
-target_link_libraries( h5minfo ${HDF5_LIBRARIES} )
-install(
-  TARGETS h5minfo
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  )
+# add_executable( h5minfo include/mhdf.h example/info.c $<TARGET_OBJECTS:mhdf> )
+# target_link_libraries( h5minfo ${HDF5_LIBRARIES} -pthread)
+# install(
+#   TARGETS h5minfo
+#   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+#   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+#   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+#   )

-add_executable( h5mvalidate example/validate.c $<TARGET_OBJECTS:mhdf> )
-install(
-  TARGETS h5mvalidate
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  )
-target_link_libraries( h5mvalidate ${HDF5_LIBRARIES} )
+# add_executable( h5mvalidate example/validate.c $<TARGET_OBJECTS:mhdf> )
+# install(
+#   TARGETS h5mvalidate
+#   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+#   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+#   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+#   )
+# target_link_libraries( h5mvalidate ${HDF5_LIBRARIES})
